main.py

```diff
@@ -0,0 +1,6 @@
+
+file = open('Murree_weather_2004_Aug.txt', 'r')

```

Comment:
Next time you send me PR, it should have more work done, and not just 6 lines of code